### PR TITLE
{lang}[GCCcore/10.3.0] Rust v1.52.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
~~requires #12755 (Python)~~ (not anymore, since we're not using Ninja anymore for building Rust)

### CODE REVIEW

File: easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://static.rust-lang.org/dist/']
+sources = ['rustc-%(version)s-src.tar.gz']
+checksums = ['3a6f23a26d0e8f87abbfbf32c5cd7daa0c0b71d0986abefc56b9a5fbfbd0bf98']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+    ('Ninja', '1.10.2'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.3.0.eb:19]:
```suggestion
```

File: easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+]
+
+configopts = "--enable-extended --sysconfdir=%(installdir)s/etc"
+
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/r/Rust/Rust-1.52.1-GCCcore-10.3.0.eb:23]:
```suggestion
# Requiring Ninja would create a circular dependency
configopts += "--set=llvm.ninja=false "

```

**Review by SebastianAchilles — COMMENTED** [review]
I suggest to remove `Ninja` as `builddependencies` for `Rust`. That removes the circular dependency.

**Comment by boegel** [comment]
Thanks for the help @SebastianAchilles!

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jrlogin12.jureca - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/77f9b829841d60a90f0d4bd3b068d473 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3128.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4c14b9c77fd2597ee6a65d83474e4b50 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3552.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/38768325c3f77348dcfed0582dbe3ba6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/c4db818104b19d02380e7d6a8a77bd2a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild2.novalocal - Linux centos linux 8.3.2011, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/91654f1a961c00e09f3f995f5d933744 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/11edecb1576d1219787158595b45da7e for a full test report.

**Review by wpoely86 — APPROVED** [review]

**Pull request merged**